packages/ui-extensions/src/surfaces/customer-account/components/DropZone/DropZone.doc.ts

```diff
@@ -0,0 +1,67 @@
+import type {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
+
+const data: ReferenceEntityTemplateSchema = {
+  name: 'DropZone',
+  description: `Dropzone lets users upload files by dragging and dropping the files into an area on a page, or activating a button. Currently dropzone does not support the ability to preview image uploads.  Developers won’t be able to use object urls directly in an image component due to extension and host running on different domains.
+    \n Any element focused within the Dropzone component, including children like the 'Add file' button, will trigger the file selector when the Enter or Spacebar is pressed.
+    `,
+  thumbnail: 'dropzone-thumbnail.png',
+  requires: '',
+  isVisualComponent: true,
+  type: '',
+  definitions: [
+    {
+      title: 'DropZoneProps',
+      description: '',
+      type: 'DropZoneProps',
+    },
+  ],
+  category: 'components',
+  defaultExample: {
+    image: 'dropzone-preview.png',
+    altText:
+      'An image showcasing the DropZone component with a button to add files with error and dragged over states.',
+    codeblock: {
+      title: 'DropZone',
+      tabs: [
+        {
+          title: 'React',
+          code: '../../../../../../ui-extensions-react/src/surfaces/customer-account/components/DropZone/examples/basic-DropZone-react.example.tsx',
+          language: 'tsx',
+        },
+        {
+          title: 'JS',
+          code: './examples/basic-DropZone-js.example.ts',
+          language: 'js',
+        },
+      ],
+    },
+  },
+  subSections: [
+    {
+      type: 'Generic',
+      anchorLink: 'best-practices',
+      title: 'Best Practices',
+      sectionContent: `
+**File storage**
+
+It's up to you to store uploaded files. You can use metafields and [the Customer Account API](https://shopify.dev/docs/api/customer/latest/mutations/metafieldsSet) to store a reference to the file alongside the relevant object.
```

Comment:
Do we have plans to create a tutorial covering how to use metafields to store image?